samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/bigqueryexport/UpdateBigQueryExport.java

```diff
@@ -0,0 +1,81 @@
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.securitycenter.snippets.bigqueryexport;
+
+// [START securitycenter_update_bigquery_export]
+
+import com.google.cloud.securitycenter.v1.BigQueryExport;
+import com.google.cloud.securitycenter.v1.SecurityCenterClient;
+import com.google.cloud.securitycenter.v1.UpdateBigQueryExportRequest;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+import java.util.UUID;
+
+public class UpdateBigQueryExport {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer): Modify the following variable values.
+    // parent: Use any one of the following resource paths:
+    //              - organizations/{organization_id}
+    //              - folders/{folder_id}
+    //              - projects/{project_id}
+    // filter: Expression that defines the filter to apply across create/update events of findings.
+    // bigQueryExportId: Unique identifier provided by the client.
+    // For more info, refer:
+    // https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query#export_findings_from_to
+    String parent = String.format("projects/%s", "your-google-cloud-project-id");
+    String filter =
+        "severity=\"LOW\" OR severity=\"MEDIUM\" AND "
+            + "category=\"Persistence: IAM Anomalous Grant\" AND "
+            + "-resource.type:\"compute\"";
+    String bigQueryExportId = UUID.randomUUID().toString().toLowerCase();
+
+    updateBigQueryExport(parent, filter, bigQueryExportId);
+  }
+
+  // Updates an existing BigQuery export.
+  public static void updateBigQueryExport(String parent, String filter, String bigQueryExportId)
+      throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      //  Set the new values for export configuration.
+      BigQueryExport bigQueryExport =
+          BigQueryExport.newBuilder()
+              .setName(String.format("%s/bigQueryExports/%s", parent, bigQueryExportId))
+              .setFilter(filter)
+              .build();
+
+      UpdateBigQueryExportRequest request =
+          UpdateBigQueryExportRequest.newBuilder()
+              .setBigQueryExport(bigQueryExport)
+              // Set the update mask to specify which properties should be updated.
+              // If empty, all mutable fields will be updated.
+              // For more info on constructing field mask path, see the proto or:
+              // https://cloud.google.com/java/docs/reference/protobuf/latest/com.google.protobuf.FieldMask
+              .setUpdateMask(FieldMask.newBuilder().addPaths("filter").build())
+              .build();
+
+      BigQueryExport response = client.updateBigQueryExport(request);
+      if (response.getFilter().equalsIgnoreCase(filter)) {
+        System.out.println("BigQueryExport updated successfully!");
+        return;
+      }
+      System.out.println("Failed to update BigQueryExport!");
```

Comment:
nit: generally it's easier to read when you put the exception case first (as it's the "non-standard" path)
```suggestion
      if (!response.getFilter().equalsIgnoreCase(filter)) {
        System.out.println("Failed to update BigQueryExport!");
        return;
      }
      System.out.println("BigQueryExport updated successfully!");
      
```

Reply:
Done. Thanks.